uint16[]  boxes      # Flattened array of bounding boxes for each face
uint16[]  marks      # Flattened array of landmark coordinates for each face
string[]  names      # List of names
float32[] scores     # List of confidence scores for every name
bool[]    genders    # List of genders
uint8[]   ages       # List of ages